observelib/observe/src/main/ballerina/src/observe/Module.md

```diff
@@ -5,7 +5,7 @@ Ballerina supports Observability out of the box. This module provides user api's
 
 To observe Ballerina code, the '--b7a.observability.enabled=true' property should be given when starting the service. 
 i.e. `ballerina run hello_world.bal --b7a.observability.enabled=true'
-For more information on Ballerina Observability visit [How to Observe Ballerina Services](https://ballerina.io/learn/how-to-observe-ballerina-code/)
+For more information on Ballerina Observability visit [How to Observe Ballerina Services](https://v1-0.ballerina.io/learn/how-to-observe-ballerina-code/).
```

Comment:
This should be https://ballerina.io/learn/how-to-observe-ballerina-code . The version should not be appended to the URL and depending on the current latest version it will get resolved automatically

Reply:
Fixed.